synapse/lib/snap.py

```diff
@@ -102,13 +102,12 @@ async def getSnapMeta(self):
 
         return meta
 
-    @contextlib.contextmanager
-    def getStormRuntime(self, opts=None, user=None):
+    @contextlib.asynccontextmanager
+    async def getStormRuntime(self, query, opts=None, user=None):
```

Comment:
why async?

Reply:
The delta originally had Runtime being an extension of Base ( and the likelyhood of that happening eventually still seems very high ) but provided we dont want to "plan for the future" in internal APIs ( which i mostly agree with ) we can back this out :)